src/ui/rendering/handlebarsrenderer.js

```diff
@@ -28,12 +28,15 @@ export default class HandlebarsRenderer extends Renderer {
     this._templates = templates || {};
   }
 
-  init (templates) {
+  init (templates, locale) {
```

Comment:
Conceptually, I think it's odd that the `HandlebarsRenderer` needs `locale`. The two concepts seem fairly removed from each other. We only have it here because this class is also responsible for creating all the HBS helpers. I think we should make an item to refactor so that we separate that responsibility out from this class. Clean Code hints at this a bit when it talks about cohesion. In this case, `locale` has low cohesion, it's just used to create the one helper. That's a sign this class is doing too much. No need to fix in this PR though! This is definitely beyond the scope of your change.